lib/deps/request-browser.js

```diff
@@ -214,9 +212,11 @@ function xhRequest(options, callback) {
 }
 
 module.exports = function(options, callback) {
-  if (typeof XMLHttpRequest === 'undefined' && !options.xhr) {
-    return fetchRequest(options, callback);
-  } else {
+  // in Chromium 41, XMLHttpRequest is an object instead of a function, so
```

Comment:
The comment is misleading, and contradicts itself with regard to the code below.  Also, it changes the behaviour so that the `fetch` fallback is used if `XMLHttpRequest` is present, but of type `'object'` instead of type `'function'`, which is probably not what we want.